polaris-core/src/testFixtures/java/org/apache/polaris/core/persistence/PolarisTestMetaStoreManager.java

```diff
@@ -828,7 +828,7 @@ void dropEntity(List<PolarisEntityCore> catalogPath, PolarisEntityCore entityToD
   }
 
   /** Grant a privilege to a catalog role */
-  void grantPrivilege(
+  public void grantPrivilege(
```

Comment:
The changes here look unrelated?

Reply:
They are not; the `EntityCacheTest` was moved into a new package and so these methods were no longer callable from it if they remained package-private